pyproject.toml

```diff
@@ -47,6 +47,7 @@ click-log = "^0.4.0"
 linkml = "^1.6.3"
 linkml-runtime = "^1.6.2"
 python = "^3.9"
+click = "^8.1.7"
```

Comment:
I don't think this is required, at least not in both the core dependencies and the dev dependencies. I don't see why we would use different version in those two groups either.

`click` is provided even without a explicit request as it is a dependency of linkml and linkml-runtime

Reply:
Removed the redundant requirement and updated the lock file